### ISSUE TITLE
Watchdog isn't aware of Geyser usage - Bedrock client on Java server

### ISSUE THREAD

**Issue by mithrandir** [doctorray117/minecraft-ondemand#58]
I'm running Spigotmc / Geysermc and the watch dog scripts cannot detect bedrock client connection. Thus my server is shutting down after 10 minutes even I'm playing and connecting. The easy way to mitigate the issue is assuming that Java server can host both of bedrock clients (UDP ports) and Java clients. Could we have such change?

PS. thanks for awesome documentation and the script. I'm AWS newbie and it only took 2 hours to setup the ECS cluster and lambda stuffs. Everything is working great except the bedrock client detection.

**Comment by yurisasc** [comment]
This is an interesting issue. I'll have a look to see if I can help with this.

**Comment by yurisasc** [comment]
@mithrandir I've opened a PR and published a test watchdog container. You can use the test container while waiting for the PR to be reviewed. You can do so by changing code: https://github.com/doctorray117/minecraft-ondemand/blob/c12ab627dc496e97c3df734a0003d5317423753c/cdk/lib/minecraft-stack.ts#L226-L232

into:
```
image: ecs.ContainerImage.fromRegistry( 
       'yurisasc/minecraft-ecsfargate-watchdog' 
     ), 
```

**Comment by Rithmatist** [comment]
> I'm running Spigotmc / Geysermc and the watch dog scripts cannot detect bedrock client connection. Thus my server is shutting down after 10 minutes even I'm playing and connecting. The easy way to mitigate the issue is assuming that Java server can host both of bedrock clients (UDP ports) and Java clients. Could we have such change?
> 
> PS. thanks for awesome documentation and the script. I'm AWS newbie and it only took 2 hours to setup the ECS cluster and lambda stuffs. Everything is working great except the bedrock client detection.

Would be easier to deploy with code :)

**Comment by mithrandir** [comment]
@yurisasc Thanks for quick change! It worked fine now, confirmed my Bedrock connection is recognized.

**Comment by doctorray117** [comment]
Hello.  Looks good, thanks for the additional testing.  @yurisasc Did you have a chance to try a fresh deployment with CDK to ensure that method worked as well?

**Comment by yurisasc** [comment]
@doctorray117 Yup. I've set up a new staging server using the image, which also works. Happy for you to review the code (or even the commit messages) if it doesn't match your format :)

**Comment by doctorray117** [comment]
@yurisasc No, it's good. Apologies for the delay.  I'm happy with the formatting and my testing time has been limited. I'll get it merged and deploy it out to the public docker repo over the weekend, and come back and close this issue afterward.

**Comment by Rithmatist** [comment]
> @yurisasc No, it's good. Apologies for the delay. I'm happy with the formatting and my testing time has been limited. I'll get it merged and deploy it out to the public docker repo over the weekend, and come back and close this issue afterward.

Do you have discord? I found a method to support modpacks and some other features

**Comment by GarrettBeatty** [comment]
any update on this?

**Comment by eNotVeryHappy** [comment]
Any update? I changed my container image to use yurisasc one and the server still closes after 10 mins
